dev-test/test-schema/flow-types.flow.js

```diff
@@ -111,7 +111,11 @@ export type ResolversParentTypes = {
 };
 
 export type QueryResolvers<ContextType = any, ParentType = $ElementType<ResolversParentTypes, 'Query'>> = {
-  allUsers?: Resolver<Array<?$ElementType<ResolversTypes, 'User'>>, ParentType, ContextType>,
+  allUsers?: Resolver<
```

Comment:
Shouldn't get modified as well, right?

Reply:
It shouldn't. same issue accured on TS resolvers.
Fixed.